01-JS-DOM-APIs/ex12.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html class="html" lang="">
+  <head>
+    <title>12</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style1-2.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+     table, th, td {
+      border: 4px solid grey;
+      text-align: center;
+    }
+</style>
+    <script src="1-2.js"> </script>
+  </head>
+
+  <body onload="Matrix()">
+    <table id="table" class="dataTables">
+      <tr>
+        <th class="tableTH">Name</th>
+        <th class="tableTH">Departure</th>
+        <th class="tableTH">Destination</th>
+      </tr>
+    </table>
+    <button onclick="importMatrix(matrix)" id="importmatrixbutton" type="buttonimportmatrix" name="buttonimportmatrix">Import</button>
```

Comment:
I see several errors here:

- `buttonimportmatrix` is not a type of button. You can find type of buttons here http://www.w3schools.com/tags/att_button_type.asp

- `onclick="importMatrix(matrix)"` pass this parameter here is wrong. You should have the matrix as a variable inside the function 

- `Import` is not a good text for the button. It is very general and could be any action